tests/test_iotedgedev_solution_build.py

```diff
@@ -69,25 +69,6 @@ def test_solution_build_and_push_with_layered_deployment():
     assert 'ERROR' not in result.output
 
 
-def test_solution_build_and_push_with_layered_deployment_no_modules():
```

Comment:
Question: Would please explain why this test is not needed?

Reply:
Please note we removed the manifest for no_modules and the related test
cases because the validation of iot edge deployment create got stricter
with the current version and otherwise the test failed.